src/Common/Infrastructure/Controllers/Attributes/CleanQueryParametersAttribute.cs

```diff
@@ -0,0 +1,31 @@
+namespace Common.Infrastructure.Controllers.Attributes
+{
+    using System;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc.Filters;
+
+    [AttributeUsage(AttributeTargets.Class, Inherited = true)]
+    public class CleanQueryParametersAttribute : Attribute, IAsyncResourceFilter
+    {
+        public async Task OnResourceExecutionAsync(ResourceExecutingContext context, ResourceExecutionDelegate next)
+        {
+            var request = context
+                .HttpContext
+                .Request;
+
+            request.QueryString = CleanQueryString(request.QueryString);
```

Comment:
is QueryString always present, or is it `null` when missing?

Reply:
checked: QueryString is non-nullable